pkgs/tools/misc/docker-slim/default.nix

```diff
@@ -0,0 +1,29 @@
+{ buildGoModule, fetchFromGitHub, stdenv, docker }:
+
+buildGoModule rec {
+  pname = "docker-slim";
+  version = "1.26.1";
+
+  goPackagePath = "github.com/docker-slim/docker-slim";
+  src = fetchFromGitHub {
+    owner = "docker-slim";
+    repo = pname;
+    rev = version;
+    sha256 = "01bjb14z7yblm7qdqrx1j2pw5x5da7a6np4rkzay931gly739gbh";
+  };
+
+  subPackages = [ "cmd/docker-slim" "cmd/docker-slim-sensor" ];
+
+  modSha256 = "05sayqbr65x98bx61ma6xnkvcsbr9g1lsviazc4szzqdizfck8yr";
+
+  meta = with stdenv.lib; {
+    description = "Minify and secure Docker containers";
+    homepage = "https://github.com/docker-slim/docker-slim";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ filalex77 ];
+    platforms = docker.meta.platforms;
```

Comment:
just curious, does it depend on docker?

Reply:
It takes images by name, so I assume it asks Docker for them.It takes images by name, so I assume it asks Docker for them.

Reply:
It takes images by name, so I assume it asks Docker for them.